doc/createAvailableOperationsListView.awk

```diff
@@ -1,14 +1,38 @@
+func print_key(key) {
```

Comment:
Unfortunately, my awk knowledge is basically non-existent. Just like CI, I also get a bunch of `return outside function body` errors. If I change `func` to `function`, it works. No idea if this is some GNU/BSD/whatever inconsistency, like different command line arguments in `grep` and friends. What ever that is worth, the GitHub highlighter also doesn't highlight the `func` keyword. My Vim, however, does (both).

    $ awk -W version
    mawk 1.3.4 20200120

Reply:
Thanks

For now, I got rid of the script completely (in https://github.com/newsboat/newsboat/pull/2161/commits/725699a5884ad30389808b7e5efea9013ce9eb0b) by placing the `kbd:[…]` straight into the input .dsv file (given that both of the generated files it is used for are asciidoc files)